test/tests/functional/pbs_multi_sched.py

```diff
@@ -580,6 +580,60 @@ def test_run_limts_per_scheduler(self):
         jc = "Not Running: User has reached server running job limit."
         self.server.expect(JOB, {'comment': jc}, id=jid4)
 
+    def test_list_multi_sched(self):
+        """
+        Test to verify that qmgr list sched works when multiple schedulers are present
+        """
+
+        self.setup_sc1()
+        self.setup_sc2()
+        self.setup_sc3()
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True)
```

Comment:
What would 'expect=True' do for a qmgr list?

Reply:
I don't know, it re-verifies the attributes?. You want me to remove it?

Reply:
verifies them against what? usually 'expect' makes sense in context of a set operation as it can verify that the attributes got updated correctly. For a list, there's nothing to verify against. So ya, I think you can remove them.